vcx/docs/getting-started/getting-started.md

```diff
@@ -0,0 +1,646 @@
+﻿# Getting Started with Libindy
+
+## A Developer Guide for Building Indy Clients Using Libindy
+
+![logo](https://raw.githubusercontent.com/hyperledger/indy-node/master/collateral/logos/indy-logo.png)
+
+* [Getting Started with Libvcx](#getting-started-with-libvcx)
+  * [What Indy and Libindy are and Why They Matter](#what-indy-and-libindy-are-and-why-they-matter)
+  * [What We'll Cover](#what-well-cover)
+  * [About Alice](#about-alice)
+  * [Infrastructure Preparation](#infrastructure-preparation)
+      * [Step 1: Getting Trust Anchor Credentials for Faber, Acme, Thrift and Government](#step-1-getting-trust-anchor-credentials-for-faber-acme-thrift-and-government)
+      * [Step 2: Connecting to the Indy Nodes Pool](#step-2-connecting-to-the-indy-nodes-pool)
+      * [Step 3: Getting the Ownership for Stewards's Verinym](#step-3-getting-the-ownership-for-stewardss-verinym)
+      * [Step 4: Onboarding Faber, Acme, Thrift and Government by the Steward](#step-4-onboarding-faber-acme-thrift-and-government-by-steward)
+        * [Connecting the Establishment](#connecting-the-establishment)
+        * [Getting the Verinym](#getting-verinym)
+      * [Step 5: Credential Schemas Setup](#step-5-credential-schemas-setup)
+      * [Step 6: Credential Definition Setup](#step-6-credential-definition-setup)
+  * [Alice Gets a Transcript](#alice-gets-a-transcript)
+  * [Apply for a Job](#apply-for-a-job)
+  * [Apply for a Loan](#apply-for-a-loan)
+  * [Explore the Code](#explore-the-code)
+
+## What Indy, Libindy and Libvcx are and Why They Matter
+
+Indy provides a software ecosystem for private, secure, and powerful identity, and libindy enables clients for it. Indy puts people — not the organizations that traditionally centralize identity — in charge of decisions about their own privacy and disclosure. Libvcx is library on top of libindy which simplifies credential exchange. This enables all kinds of rich innovation: connection contracts, revocation, novel payment workflows, asset and document management features, creative forms of escrow, curated reputation, integrations with other cool technologies, and so on.
```

Comment:
We can add sentence that libindy is a low level library that provides fine configuration, but vcx is more high-level library concentrated on hiding low-level details and increasing of application development efficiency.

Reply:
Done